src/test/resources/pacts/publicapi-ledger-search-payment-with-charge-in-success-state.json

```diff
@@ -84,13 +84,13 @@
           ],
           "_links": {
             "self": {
-              "href": "http://ledger.service.backend/v1/transaction?account_id=123456&state=SUCCESS&transaction_type=PAYMENT&page=1&display_size=500"
+              "href": "http://ledger.service.backend/v1/transaction?account_id=123456&state=success&transaction_type=PAYMENT&page=1&display_size=500"
```

Comment:
the changes in this file from `SUCCESS` to `success` weren't meant to be here but the PR was failing because of this. this is because the test is searching the state using `"state": ["success"]` and something must have changed in ledger so the case is reflected in the hrefs.

Reply:
🤔 not entirely sure I understand

Reply:
so because in the request we're searching by state "success" and not "SUCCESS", the hrefs returned in the `_links` will be `http://ledger.service.backend/v1/transaction?account_id=123456&state=success` and not `http://ledger.service.backend/v1/transaction?account_id=123456&state=SUCCESS`. i'm not sure why this doesn't work all of a sudden but I don't have the wherewithal to investigate.